src/framework/script/script-attributes.js

```diff
@@ -148,6 +146,50 @@ function rawToValue(app, args, value, old) {
     return value;
 }
 
+/**
+ * @typedef {Object} AttributeSchema
+ * @property {"boolean"|"number"|"string"|"json"|"asset"|"entity"|"rgb"|"rgba"|"vec2"|"vec3"|"vec4"|"curve"} type - The Attribute type
+ * @property {boolean} [array] - True if this attribute is an array of `type`
+ */
+
+/**
+ * Takes an attribute schema, a value and current value, and return a new value
+ *
+ * @param {import('../../framework/application.js').Application} app - The working application
+ * @param {AttributeSchema} schema - The attribute schema used to resolve properties
+ * @param {*} value - The raw value to create
+ * @param {*} current - The existing value
+ * @returns {*} The return value
+ */
+export function attributeToValue(app, schema, value, current) {
```

Comment:
Just wanted to check...have you taken a look at the generated API reference to see whether any JSDoc blocks are unexpectedly being made public? Hard to tell if, say, this one is...

Reply:
Yep, this no longer needs to be public. Removing the export